### PR TITLE
feat(eslint): sort import rule

### LINKED ISSUE #7023: [Infra]: add eslint plugin that sorts imports correctly
#### Summary
Would be nice with an eslint plugin that sorts imports based on their name, import source and parameters.

### PR DESCRIPTION
**Pull request by evenstensberg**
Closes #7023

### Describe your changes:
Rule to sort imports.

https://github.com/lydell/eslint-plugin-simple-import-sort

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [x] Tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Reviewer has tested changes by following the provided instructions?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate automated tests included?
* [ ] Code style and in-line documentation are appropriate?
* [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

**Comment by evenstensberg** [comment]
This should be added after UMD -> ESM

**Comment by ozyx** [comment]
> This should be added after UMD -> ESM

Do you think it would cause us any issues if we add it before this is completed? This is great and I would like to incorporate it into our eslint config.

File: package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
     "eslint-plugin-compat": "4.1.4",
     "eslint-plugin-playwright": "0.12.0",
     "eslint-plugin-prettier": "4.2.1",
+    "eslint-plugin-simple-import-sort": "^10.0.0",
```
Comment (ozyx) [inline package.json:32]: Let's go ahead and pin this to the latest version so dependabot can manage it

**Review by ozyx — CHANGES_REQUESTED** [review]

**Comment by evenstensberg** [comment]
@ozyx You can run the plugin without any issues (se eslintrc for rules to toggle). This plugin is very simple, as it says. 

I'd rather have a look at  https://github.com/import-js/eslint-plugin-import if that is ok for you? It's more extensive and checks more than just the import order

**Comment by evenstensberg** [comment]
@ozyx PTAL! Atm I've configured to only have warnings on import/export rules. You can fix those pretty easily.

**Review by ozyx — APPROVED** [review]
Good stuff. Thank you!

**Pull request merged**